share/mdnjs/fetch.sh

```diff
@@ -1,6 +1,15 @@
 #!/bin/bash
 python fetch.py --cachedir downloads \
-                --cachejournal ".cachejournal" \
+                --cachejournal ".cachejournal_api" \
+                --patt "en-US/docs/Web/API" \
+                --sitemap http://developer.mozilla.org/sitemaps/en-US/sitemap.xml \
+                --sleep 5
+
+python fetch.py --cachedir downloads \
+                --cachejournal ".cachejournal_js" \
                 --patt "en-US/docs/Web/JavaScript/Reference/Global_Objects" \
                 --sitemap http://developer.mozilla.org/sitemaps/en-US/sitemap.xml \
                 --sleep 5
+
+cat .cachejournal_api >> .cachejournal
+cat .cachejournal_js >> .cachejournal
```

Comment:
The idea here is that we have to give it a base url that it uses to grab the individual pages.  We want Web/Api and Global which have different urls but a single cache journal.  So run fetch twice with each url and create a single cachejournal at the end.